FAIL: TestFileWmf.test_load_raw on Windows

On Windows 10 1607:
```
--------------------------------------------------------------------
PIL SETUP SUMMARY
--------------------------------------------------------------------
version      Pillow 3.5.0.dev0
platform     win32 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40)
             [MSC v.1500 64 bit (AMD64)]
--------------------------------------------------------------------
--- JPEG support available
--- OPENJPEG (JPEG2000) support available (2.1)
--- ZLIB (PNG/ZIP) support available
*** LIBIMAGEQUANT support not available
--- LIBTIFF support available
--- FREETYPE2 support available
--- LITTLECMS2 support available
--- WEBP support available
--- WEBPMUX support available
--------------------------------------------------------------------
To add a missing option, make sure you have the required
library and headers.
See https://pillow.readthedocs.io/en/latest/installation.html#building-from-source

To check the build, run the selftest.py script.

--------------------------------------------------------------------
Pillow 3.5.0.dev0 TEST SUMMARY
--------------------------------------------------------------------
Python modules loaded from D:\Build\Pillow\Pillow-git\PIL
Binary modules loaded from D:\Build\Pillow\Pillow-git\PIL
--------------------------------------------------------------------
--- PIL CORE support ok
--- TKINTER support ok
--- FREETYPE2 support ok
--- LITTLECMS2 support ok
--- WEBP support ok
--- JPEG support ok
--- OPENJPEG (JPEG2000) support ok
--- ZLIB (PNG/ZIP) support ok
--- LIBTIFF support ok
--------------------------------------------------------------------
Running selftest:
--- 57 tests passed.
<snip>
======================================================================
FAIL: TestFileWmf.test_load_raw
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\Build\Pillow\Pillow-git\Tests\test_file_wmf.py", line 27, in test_load_raw
    self.assert_image_similar(im, imref, 0.5)
  File "D:\Build\Pillow\Pillow-git\Tests\helper.py", line 110, in assert_image_similar
    ave_diff, epsilon))
AssertionError:  average pixel value difference 1.8203 > epsilon 0.5000
-------------------- >> begin captured logging << --------------------
PIL.PngImagePlugin: DEBUG: STREAM 'IHDR' 16 13
PIL.PngImagePlugin: DEBUG: STREAM 'IDAT' 41 20247
PIL.PngImagePlugin: DEBUG: STREAM 'IHDR' 16 13
PIL.PngImagePlugin: DEBUG: STREAM 'IDAT' 41 474
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 817 tests in 14.297s

FAILED (SKIP=23, failures=1)
```

--- a/Tests/test_file_wmf.py
+++ b/Tests/test_file_wmf.py
@@ -24,7 +24,7 @@ class TestFileWmf(PillowTestCase):
             # Compare to reference rendering
             imref = Image.open('Tests/images/drawing_wmf_ref.png')
             imref.load()
-            self.assert_image_similar(im, imref, 0.5)
+            self.assert_image_similar(im, imref, 2.0)
 
 
 if __name__ == '__main__':

